traceutil.go

```diff
@@ -3,14 +3,26 @@ package main
 import (
 	"context"
 	"encoding/base64"
+	"errors"
 
 	"go.opencensus.io/trace"
 	"go.opencensus.io/trace/propagation"
 )
 
+// TraceConfig is the struct used to pass configuration to the tracer
+type TraceConfig struct {
```

Comment:
Are we just going to end up redefining `go.ioencensus.io/trace.Config` here?  I'm not convinced we need a new TraceConfig struct.

Generally, we should take all of the inputs (flags, etc), put them in a Config struct, and then define a validate function on that config struct.

Or better yet, [use a ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) to hold this information so we could write patches for individual fields, and don't have to use any flags other than the location of the config file.

Reply:
done